app/sentry.ts

```diff
@@ -24,3 +31,25 @@ Sentry.init({
     },
     ...config.sentry
 });
+
+Sentry.configureScope(function(scope) {
+    function addAdapterError(error: any, event: Sentry.Event) {
+        if (error.isAdapterError) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5114150)